src/parser/parsers/heroes.py

```diff
@@ -197,4 +197,4 @@ def _calc_bullet_velocity(self, spline):
                 raise Exception('Bullet speed curve is not constant')
 
         # If constant, return the y
-        return last_y / ENGINE_UNITS_PER_METER
+        return int(round(last_y / ENGINE_UNITS_PER_METER, 0))
```

Comment:
hes back

Reply:
lol how does it do this, it never prompted me in merge conflict

Reply:
It should overwrite when you merge in master you'd think 🤔